extensions/amp-ad-network-base/0.1/amp-ad-context.js

```diff
@@ -0,0 +1,288 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class AmpAdContext {
```

Comment:
Pruning this file now. All these values are things that rendering methods in amp-a4a currently return or set, but not all of these will be needed.

Reply:
I was imagining a simple `context` object in JSON, so that any new impl can add their own stuff.
Most of the fields in this class should either go to fields of the base class, or renderer/validator params.

Reply:
I've removed AmpAdContext and references too it, which should simplify things a bit.